### PR TITLE
Add landing web page

### LINKED ISSUE #10: Add landing web page

### PR DESCRIPTION
**Pull request by Shalevro2**
Create board.html file- landing web page

Create projboard directory by < pipenv run python manage.py startapp projboard > command

Modify setting.py- add our app to INSTALLED_APPS

Modify urls.py- add the board to urlpatterns

**Note:** Most changes had been auto-generated.

Resolve #10

### CODE REVIEW

**Review by Yarboa — DISMISSED** [review]
LGTM

**Comment by Yarboa** [comment]
> ~~LGTM~~

Sorry Dismissed my approval, please fix CI errors

**Review by Yarboa — CHANGES_REQUESTED** [review]
Sorry Dismissed my approval, please fix CI errors

File: projboard/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (rawadabu) [inline projboard/tests.py]:
```
# from django.test import TestCase 
```

File: projboard/admin.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (rawadabu) [inline projboard/admin.py]:
```
# from django.contrib import admin
```

File: projboard/models.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from django.db import models
```
Comment (rawadabu) [inline projboard/models.py]:
```
# from django.db import models
```

File: projboard/models.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+from django.utils import timezone
```
Comment (rawadabu) [inline projboard/models.py]:
```
# from django.utils import timezone
```

**Review by rawadabu — COMMENTED** [review]
Changes to pass the CI

**Comment by Shalevro2** [comment]
> 

Hi Rawad, 
Thank you about the review, I added a new commit a few minute ago.

File: projboard/tests.py (hunk 1 above)
Comment (Yarboa) [inline projboard/tests.py]: Please comment or remove this line,
We are not using TestCases for testing.

**Review by Yarboa — COMMENTED** [review]
Sorry Dismissed my approval, please fix CI errors

File: projboard/tests.py (hunk 1 above)
Comment (Shalevro2) [reply projboard/tests.py]: I already comment this line in the second commit (Fix ci errors)

**Review by Yarboa — APPROVED** [review]

**Pull request merged**